vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -325,7 +325,7 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
 
             globalEventStream.post(new DotnetAcquisitionRequested(commandContext.version, commandContext.requestingExtensionId ?? 'notProvided', commandContext.mode!, commandContext.installType ?? 'global'));
 
-            const existingOfflinePath = await getExistingInstallOffline(worker, workerContext);
+            const existingOfflinePath = await getExistingInstallIfOffline(worker, workerContext);

```

Comment:
Is the old method still being used? I noticed the code for `getExistingInstallOffline` still exists